File: 백준/Bronze/25304. 영수증/영수증.java
import java.util.Scanner; 
public class Main{
    public static void main(String args[]){
        Scanner in = new Scanner(System.in);
        int X = in.nextInt();
        int N = in.nextInt();
        int sum=0,total=0;
        int a=0,b=0; 
        for(int i=1; i<=N; i++){
            a=in.nextInt();
            b=in.nextInt();
            total += a*b;
        }
        if(total==X) System.out.println("Yes");
        else System.out.println("No");
        
    }
}

File: 백준/Bronze/25304. 영수증/영수증.c
#include <stdio.h>
int main(void){
    int X=0,N=0,a=0,b=0,sum=0;
    scanf("%d",&X);
    scanf("%d",&N);
    for(int i=1; i<=N; i++){
        scanf("%d",&a);
        scanf("%d",&b);
        sum += a*b;
    }
    if(sum==X) printf("Yes");
    else printf("No");
}